myScript.js

```diff
@@ -0,0 +1,73 @@
+const bookList = document.getElementById('bookListItems');
+const form = document.getElementById('form');
+const removebtn = document.getElementsByClassName('removebtn');
+const newFormBook = document.getElementById('newFormBook');
+const newFormAuthor = document.getElementById('newFormAuthor');
+let books = [];
+const findBooks = () => {
+  if (localStorage.getItem('bookList') !== []) {
+    books = JSON.parse(localStorage.getItem('bookList'));
+  }
```

Comment:
- [ ] Please, I am getting this 👇  error message from my console. I guess the reason is because you cannot iterate over an empty array in the **first instance.**
```suggestion
  if (localStorage.getItem('bookList')) {
    books = JSON.parse(localStorage.getItem('bookList'));
  **}**
```

## Error message
![png-transparent-nigerian-naira-currency-symbol-symbol-miscellaneous-angle-text](https://user-images.githubusercontent.com/103386183/211585022-cd2244ce-aa6d-444a-936a-d383436f5262.png)

Reply:
Hey @OyePriscilla , 
 Thanks a lot for the review. I however don't understand the response clearly. I don't think we are iterating over an empty array as you've suggested. The code is not also with any error in both of our computers. Could it rather be a problem with your computer?

Reply:
Hey @OyePriscilla ,
Thanks a lot for the review. I however don't understand the response clearly. I don't think we are iterating over an empty array as you've suggested. The code is not also with any error in both of our computers. Could it rather be a problem with your computer?